README.md

```diff
@@ -313,6 +324,52 @@ ssv-dkg init --configPath ./initiator-config/initiator.yaml
 
 If the `--configPath` parameter is not provided, `ssv-dkg` will be looking for a file named `config.yaml` in `./config/` folder at the same root as the binary (i.e. `./config/config.yaml`)
 
+#### Key resharing
+
+Using DKG tool is possible to reshare existing validator key to a new set of operators. For example, at initial DKG we created a validator with [1,2,3,4] operator shares, now we can use this operators to reshare this validator to a new set of disjoint operators [5,6,7,8] or to a joint set [1,2,5,6] etc. 
+
+⚠️ Note: all operators (ols set and new set) should be online to complete the resharing protocol. 
+
+⚠️ Note: new threshold will be computed based on new set of operators using 3f+1 tolerance. 
+
+The Initiator creates the initial details needed to run DKG between all operators via the init command. You can launch the following command with the appropriate values to each parameter:
+```sh
+ssv-dkg reshare \
+          --operatorIDs 1,2,3,4 \
+          --newOperatorIDs 5, 6, 7, 8 \
+          --oldID "dbd12b3155454666a6710a2262695bb82cda41948d612d98" \
+          --operatorsInfoPath ./examples/operators_integration.json \
+          # Alternatively:
+          # --operatorsInfo: '[{"id": 1,"public_key": "LS0tLS1CRUdJTiBSU0....","ip": "http://localhost:3030"}, {"id": 2,"public_key": "LS0tLS1CRUdJTiBSU0....","ip": "http://localhost:3030"},...]'
+          --owner 0x81592c3de184a3e2c0dcb5a261bc107bfa91f494 \
+          --nonce 4 \
+          --network "mainnet" \
+          --outputPath /output \
+          --initiatorPrivKey ./encrypted_private_key.json \
+          --initiatorPrivKeyPassword ./password \
```

Comment:
we should probably add a note about that generating a new key here isn't possible since we want to use the previous key used to init.

Reply:
Agree. Ill add a note to readme.